src/reducers/auth.js

```diff
@@ -0,0 +1,41 @@
+import api from 'utils/api';
+
+const LOGIN_START = 'rmp/users/LOGIN_START';
+const LOGIN_SUCCESS = 'rmp/users/LOGIN_SUCCESS';
+const LOGIN_FAILURE = 'rmp/users/LOGIN_FAILURE';
+
+const initialState = {
+  isLoggedIn: false,
+  isLoggingIn: false,
+  error: '',
```

Comment:
would prefer null

Reply:
Error will probably be an object so empty string might be misleading here.

Reply:
Even though `null === false && '' === false`